scripts/install/git.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+git config --global diff.renames true
```

Comment:
Why are these global settings? We do not need to be overwriting a **global** Git config without **VERY** good reason, and we need to tell them about it loud and clear. Can these be changed to a `user`, `local`, or even `system` scope? And this script needs to print out info about what's being set at what level.

Reply:
pretty simple: if a repo moves and credentials are not cached, all platforms prompt the facade worker for a login, basically stopping the process.

Reply:
@ccarterlandis : Large repos need large renames numbers.

Reply:
I understand why we need the settings, but I still don't understand why these need to be **global** settings. If they HAVE to be global, then this is not going in an automated script. Changing global Git settings with out a user's consent or knowledge is a really quick way to piss them off :)

Reply:
Well, if they are not "global" they have to be set on every repository... or am I being `git dumb`?